_es/tour/basics.md

```diff
@@ -6,4 +6,321 @@ discourse: false
 
 partof: scala-tour
 language: es
+
+num: 2
+next-page: unified-types
+previous-page: tour-of-scala
 ---
+
+En esta página, cubriremos lo básico de Scala.
+
+## Probando Scala en el navegador
+
+Puedes ejecutar Scala en tu navegador con ScalaFiddle.
+
+1. Ve a [https://scalafiddle.io](https://scalafiddle.io).
+2. Pega `println("Hello, world!")` en el panel izquierdo.
+3. Presiona el botón de "Run". El resultado aparece en el panel de la derecha.
+
+Esta es una forma simple y con cero configuración de experimentar con piezas de código en Scala.
+
+Many of the code examples in this documentation are also integrated with ScalaFiddle, so you
+can directly experiment with them simply by clicking the Run-button.
+Muchos de los ejemplos de código en esta documentación están integrados también con ScalaFiddle, para que puedas experimentar directamente con ellos simplemente presionando el botón de "Run".
+
+## Expresiones
+
+En Scala todo es una expresión (cada sentencia "devuelve" un valor).
+
+```tut
+1 + 1
+```
+
+Se puede ver el resultado de evaluar expresiones usando `println`.
+
+{% scalafiddle %}
+```tut
+println(1) // 1
+println(1 + 1) // 2
+println("Hello!") // Hello!
+println("Hello," + " world!") // Hello, world!
+```
+{% endscalafiddle %}
+
+## Valores
```

Comment:
This is a tricky case of translation. "Valor" would usually mean the speific _value_, such as `3`, `False`, `"Cave Lupum"`, etc; and there is no right translation. 

In a pure functional language, like Haskell, it would be call "Local Variable", but then in Scala you mistake it with a `var`.

Reply:
Tal vez "devuelve un resultado"? En contraste con Java, que ' { String x= "a"; }` no devuelve nada...

Reply:
Volvi a la traduccion de lo que dice el basic.md en ingles : Expressions are computable statements.

Reply:
>"Valor" would usually mean the speific value, such as 3, False, "Cave Lupum", etc; and there is no right translation.

Creo q `Valores` está bien. Llevando a inglés la frase de @diesalbla: "the _value_ is `3`, `False`,...". Creo que me suena parecido y dejaría `Valor/es`.

Reply:
Quizás fuera mejor llamarlas variables mutables o inmutables.

Reply:
Si, creo que he visto "variables inmutables" pero parece un contrasentido :smile: 
No me molesta dejarlo en "Valores" y mas adelante cambiarlo si encontramos una traduccion mejor